app/models/labware_creators/pooled_wells_by_sample.rb

```diff
@@ -0,0 +1,102 @@
+# frozen_string_literal: true
+
+require_dependency 'labware_creators/base'
+
+module LabwareCreators
+  # This labware creator pools PBMC isolations with the same samples in pairs
+  # (or configured number of source wells) per destination before cell counting
+  # to reduce the number runs on Celleca (cell counting). The robot transfer is
+  # done from LRC Blood Bank to LRC PBMC Bank plate. Only the wells with passed
+  # state will be transferred to the destination plate. The destination wells
+  # are compressed to top left by column on the plate.
+  class PooledWellsBySample < Base
```

Comment:
I think we need a more descriptive name for this creator that suggests the grouping in pairs / a number.
Maybe PooledWellsBySampleInGroups ? PooledWellsBySampleGrouped ? PooledWellsBySampleInSets ? PooledWellsBySampleInLots ?

Reply:
More specific and descriptive name should be given. PooledWellsBySampleInGroups can be used unless any other can be found.